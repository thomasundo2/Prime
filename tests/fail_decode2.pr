int main()
{
    lint x;
    x = decode(67l);
    return 0;
}