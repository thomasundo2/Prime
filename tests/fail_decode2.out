Fatal error: exception Failure("illegal assignment lint = string in x = decode(67)")
